python/ray/tests/test_generators.py

```diff
@@ -247,6 +292,26 @@ def generator(self, num_returns, store_in_plasma):
         g = Generator.remote()
         remote_generator_fn = g.generator
 
+    elif execution == "async_actors":
```

Comment:
Can we add a unit test if the execution is actually async? E.g., submit 10 of asyncio.sleep(1) and make sure the result is returned within 1.5 seconds or sth like that